### ISSUE TITLE
Automated workflow for semantic releases

### ISSUE THREAD

**Issue by mrcjkb** [nvim-orgmode/orgmode#828]
### Does this feature exist in Emacs orgmode core?

N/A

### Feature value

The last release is a few months old. It would me nice if new releases with fixes could be published more frequently.
Since you already use conventional commits, perhaps an automated workflow with [the release-please-action](https://github.com/googleapis/release-please-action) would make it easier to publish releases more frequently?

### Additional context

I'd be happy to contribute a workflow if you're open to it :smile:

**Comment by ficcdaf** [comment]
I support this certainly. Release-please is very easy to set up.

**Comment by kristijanhusak** [comment]
Yeah, we can add this, just please set it up as a `workflow_dispatch`. I don't want to make a new release on every push to master branch.

**Comment by kristijanhusak** [comment]
[workflow_dispatch](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch) is basically a button in the actions that will trigger the workflow.

The PR solution is also ok, so you can go with that if you want.

**Comment by ficcdaf** [comment]
OK, should I also add a note under the contribution guidelines to please use conventional commits?

**Comment by kristijanhusak** [comment]
Yeah, you can do that. Set up a PR and we can discuss any necessary details there.

**Comment by mrcjkb** [comment]
> OK, should I also add a note under the contribution guidelines to please use conventional commits?

I would suggest a checklist in a PR template. 
If you squash merge PRs, it's enough for the PR title to have a conventional commit prefix.

**Comment by ficcdaf** [comment]
I created the PR. I don't know how to request a review on it (or if that's something I'm supposed to do?) I'm still new to this contributing thing. 

Anyways, I added the workflow, added a PR template, and updated `contributing.org`. Let me know if I should change anything!
